include/nbl/builtin/hlsl/emulated_float64_t.hlsl

```diff
@@ -0,0 +1,630 @@
+#include <nbl/builtin/hlsl/cpp_compat.hlsl>
+
+#ifdef __HLSL_VERSION
+#define LERP lerp
+#else
+#define LERP nbl::hlsl::lerp
+#endif
+
+#ifdef __HLSL_VERSION
+#define ABS abs
+#else
+#define ABS std::abs
+#endif

```

Comment:
Shouldn't these be somewhere in cppcompat headers?

Reply:
ok it's already mentioned in  https://github.com/Devsh-Graphics-Programming/Nabla/pull/696/files#r1628389265 :D

Reply:
why did you mark as resolved?

Reply:
done